tests/functional/testSubscriptionNeeded.js

```diff
@@ -0,0 +1,172 @@
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ const assert = require('assert');
+ const vpn = require('./helper.js');
+ const queries = require('./queries.js');
+ const { startAndConnect } = require('./setupVpn.js');
+
+describe("subscription needed tests", function() {
+  // TODO: Create tests for this screen.
+
+  describe('subscription needed related telemetry tests', function() {
+    if(vpn.runningOnWasm()) {
+      return;
+    }
+
+    this.ctx.guardianOverrideEndpoints = {
+      POSTs: {
+        '/api/v2/vpn/login/verify': {
+          status: 200,
+          body: {
+            user: {
+              avatar: '',
+              display_name: 'Test',
+              email: 'test@mozilla.com',
+              max_devices: 5,
+              // This is the important bit.
+              // The user data has no active subscriptions.
+              subscriptions: {},
```

Comment:
For a user with no active subscription, should this be changed to the following instead?

```js
      subscriptions: {vpn: {active: false}},
```

Reply:
I think both result in the same effect? cc @jswinarton just in case. I think the difference between the two is that one is a user that has never subscribed and the other a user that was subscribed before and the subscription expired.